hs_modflow_modelinstance/models.py

```diff
@@ -589,26 +589,13 @@ def metadata(self):
         md = MODFLOWModelInstanceMetaData()
         return self._get_metadata(md)
 
-    def has_required_content_files(self):

```

Comment:
Not sure how you don't need this function. This function is used for testing if the resource can be set public or not.